skbeam/core/tests/test_correlation.py

```diff
@@ -181,69 +190,95 @@ def test_auto_corr_scat_factor():
 
     g2 = auto_corr_scat_factor(lags, beta, relaxation_rate, baseline)
 
-    assert_array_almost_equal(g2, np.array([1.5, 1.0, 1.0, 1.0, 1.0,
-                                            1.0, 1.0, 1.0]), decimal=8)
+    assert_array_almost_equal(
+        g2, np.array([1.5, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]), decimal=8
+    )
 
 
 def test_bad_images():
     setup()
-    g2, lag_steps = multi_tau_auto_corr(4, num_bufs,
-                                        rois, img_stack)
+    g2, lag_steps = multi_tau_auto_corr(4, num_bufs, rois, img_stack)
     # introduce bad images
     bad_img_list = [3, 21, 35, 48]
     # convert each bad image to np.nan array
     images = bad_to_nan_gen(img_stack, bad_img_list)
 
     # then use new images (including bad images)
-    g2_n, lag_steps_n = multi_tau_auto_corr(4, num_bufs,
-                                            rois, images)
+    g2_n, lag_steps_n = multi_tau_auto_corr(4, num_bufs, rois, images)
 
     assert_array_almost_equal(g2[:, 0], g2_n[:, 0], decimal=3)
     assert_array_almost_equal(g2[:, 1], g2_n[:, 1], decimal=3)
 
 
 def test_one_time_from_two_time():
+    np.random.seed(333)
     num_lev = 1
     num_buf = 10  # must be even
     x_dim = 10
     y_dim = 10
     stack = 10
     imgs = np.random.randint(1, 3, (stack, x_dim, y_dim))
+    for i in range(imgs.shape[0]):
```

Comment:
I think this could be accomplished as `imgs[:, 0, 0] += 5`

Reply:
ok.  will work on that.

Reply:
@dmgav your suggestion is better. thanks for the kind review.
 Just a general git etiquette question:  once the tests pass, do I click resolve conversation or do one of you?